pkg/didcomm/dispatcher/outbound.go

```diff
@@ -22,6 +22,12 @@ import (
 	"github.com/hyperledger/aries-framework-go/pkg/vdr/fingerprint"
 )
 
+/* const (
+	legacyMediaType			 = "JWM/1.0"
+	didCommV1MediaType       = "application/didcomm-enc-env"
+	didCommV2MediaType       = "application/didcomm-encrypted+json"
+) */
+
```

Comment:
Did you intend to keep this commented out code here?

Reply:
yes. this is not ready to be used for now, it will be uncommented when encryption keys and packer selection based on MediaType is done, in a future pr.